### PR TITLE
Remove blue :focus styling from chrome

### LINKED ISSUE #3167: Wheel zoom tool on chrome
From @bryevdv on #3163 

> ping @birdsarah I am having trouble turning on the wheel zoom by clicking the button. It's not working at all on chrome for me, it just leaves this weird persistent highlight:

screen shot 2015-11-18 at 4 16 03 pm

### PR DESCRIPTION
**Pull request by birdsarah**
Fixes #3167 (at least the blue bit - the tools are working fine for me)

### CODE REVIEW

**Comment by mattpap** [comment]
For me it's gray bit, but this changes removes the issue altogether.

File: bokehjs/src/less/main.less [hunk 1]
```diff
@@ -148,3 +148,7 @@
   padding-left: 50px;
   padding-bottom: 10px;
 }
+
+.bk-toolbar-button.hover:focus {
```
Comment (mattpap) [inline bokehjs/src/less/main.less]: `hover` is bokeh's class? If so, it should get `bk` prefix in another PR.

**Comment by birdsarah** [comment]
@mattpap, you marked this accepted - want to merge?

**Comment by mattpap** [comment]
> you marked this accepted - want to merge?

Yeah, just giving some time to the team.

**Pull request merged**